frontend/src/config/rbd/layers.json

```diff
@@ -3178,14 +3181,28 @@
     "title": "Phase classification",
     "type": "admin_level_data",
     "path": "data/rbd/ch/admin2_{YYYY_MM_DD}.json",
-    "dates": ["2018-02-01", "2018-06-01", "2018-10-01", "2019-02-01", "2019-06-01", "2019-10-01", "2020-02-01", "2020-06-01", "2020-10-01", "2021-02-01", "2021-06-01", "2021-10-01", "2022-02-01", "2022-06-01", "2022-10-01", "2023-06-01"],
+    "dates": [
+      "2018-02-01",
+      "2018-06-01",
+      "2018-10-01",
+      "2019-02-01",
+      "2019-06-01",
+      "2019-10-01",
+      "2020-02-01",
+      "2020-06-01",
+      "2020-10-01",
+      "2021-02-01",
+      "2021-06-01",
+      "2021-10-01",
+      "2022-02-01",
+      "2022-06-01",
+      "2022-10-01"
```

Comment:
Is it normal that there is so much changes in the rdb layer ? For instance here we loose the most recent data point "2023-06-01"

Reply:
Nice catch! no it's not normal. I must have pushed some conflicting changes from a different local branch. I'll revert those